README.md

```diff
@@ -18,8 +18,8 @@ And here is our best guess on how to effectively make a submission for the [full
 
 The SNARK prover is composed of several FFTs and multiexponentiations. In the C++ reference implementation, the [FFTs are here](https://github.com/CodaProtocol/snark-challenge-prover-reference/blob/master/libsnark/main.cpp#L90) and the [multiexponentiations are here](https://github.com/CodaProtocol/snark-challenge-prover-reference/blob/master/libsnark/main.cpp#L201).
 
-1. Once you've finished the tutorial, try swapping out the multi-exponentiations in the prover for on-gpu versions using the curve operations from the tutorial. This may already lead to a big speed improvement. 
-2. Do the multi-exponentiation component entirely on-gpu, using an on-gpu [reduce](https://github.com/NVIDIA/cuda-samples/tree/master/Samples/reduction)
+1. Once you've finished the tutorial, try improving the multi-exponentiations with on-gpu versions using the curve operations from the tutorial. It may be easier to first put only the exponentiations on the gpu, and sum up all of the exponentiations on cpu. This may already lead to a big speed improvement. 
```

Comment:
"sum up the resulting values". Also GPU and CPU should be capitalized everywhere.

Reply:
I switched to your version - will capitalize cpu + gpu everywhere